### PR TITLE
[10.0] [MIG] partner_identification

### PR DESCRIPTION
**Pull request by jjscarafia**

### CODE REVIEW

**Comment by rafaelbn** [comment]
Hello @jjscarafia , please could you check travis? Thanks!

File: partner_identification/__manifest__.py [hunk 1]
```diff
@@ -22,10 +22,10 @@
         'security/ir.model.access.csv',
     ],
     'author': 'ChriCar Beteiligungs- und Beratungs- GmbH, '
-              'Antiun Ingeniería S.L.',
+              'Antiun Ingeniería S.L.,'
```
Comment (rafaelbn) [inline partner_identification/__manifest__.py]: Please change to the new name "Tecnativa"

**Review by rafaelbn — COMMENTED** [review]
Still pending functional test

**Comment by jjscarafia** [comment]
@rafaelbn done! thanks

**Review by rafaelbn — APPROVED** [review]
Tested in runbot

File: partner_identification/__manifest__.py [hunk 2]
```diff
@@ -11,19 +11,22 @@
 {
     'name': 'Partner Identification Numbers',
     'category': 'Customer Relationship Management',
-    'version': '9.0.1.0.0',
+    'version': '10.0.1.0.0',
+    'depends': [
+        'sales_team',
```
Comment (yajo) [inline partner_identification/__manifest__.py:16]: Can't you put the settings view in another place under _Sales > Configuration_ that does not require this dependency?

Not a blocker, though, but would be nice.

**Review by yajo — APPROVED** [review]

**Comment by tarteo** [comment]
Hi @jjscarafia please update the "Try me on Runbot" button to 10.0 and use <odoo> instead of <openerp><data>.

**Comment by jjscarafia** [comment]
@tarteo done
@Yajo as far as I can see the hole sales menu is created by sales_team module.

**Comment by yajo** [comment]
Anyway I just saw that [`sale` depends on `sales_team` and not the other way around](https://github.com/odoo/odoo/blob/10.0/addons/sale/__manifest__.py#L44), so not a big problem.

**Pull request merged**